sdk/keyvault/keyvault-keys/test/public/utils/common.ts

```diff
@@ -4,7 +4,11 @@
 import { SupportedVersions, supports, TestFunctionWrapper } from "@azure/test-utils";
 import { env } from "@azure-tools/test-recorder";
 import { assert } from "chai";
-import { LATEST_API_VERSION } from "../../src/keysModels";
+
+/**
+ * The latest supported Key Vault service API version
+ */
+export const LATEST_API_VERSION = "7.3-preview";
```

Comment:
Same here - can we infer this from https://github.com/Azure/azure-sdk-for-js/pull/19959/files#diff-ab4bd88f9b38991b39675497b587e03bcd84e79b8d7312509188616c52cfec7bR51

Reply:
What do you mean by inferring?

Reply:
Maor means this: https://www.typescriptlang.org/play?#code/FAYw9gdgzgLgBFApgJwG4EsSIGoquyKOAXjgG0AiAdgDoAGCgGjmpoEYmXaAmT1gZgC0AB2SIMiAO4UAunACGRcNBgBuUIXgAZAIIAVAKIBlPQH0dABQCSp7AYBKRqwHkAciQQoMWXMnyEyJDRMHDwCaBoAG0QIAHMYAAs4QTg2GWANaDBoqLBYgApdQxNza1sHJzcASiA (Maor shared this with me over Teams)